react/features/overlay/actionTypes.js

```diff
@@ -0,0 +1,25 @@
+import { Symbol } from '../base/react';
+
+/**
+ * The type of the Redux action which signals that a suspend was detected.
+ *
+ * {
+ *     type: SUSPEND_DETECTED
+ * }
+ * @public
```

Comment:
After the introduction of `_` as a access modifier hint, we left all actionTypes without `_` which means that they're public. For the sake of consistency, people may feel that `@public` is necessary so I'm OK with it. It should be noted though that `@public` is missing pretty much everywhere and not only on actionTypes.